Gems/LyShine/Code/Source/UiParticleEmitterComponent.cpp

```diff
@@ -1022,6 +1022,51 @@ void UiParticleEmitterComponent::SetOverrideAlpha(float alpha)
     m_isAlphaOverridden = true;
 }
 
+void UiParticleEmitterComponent::SetImageIndex(AZ::u32 index)
+{
+    if (m_spriteSheetCellIndex != index)
+    {
+        m_spriteSheetCellIndex = index;
+        MarkRenderGraphDirty();
+    }
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Nit: Can omit these comment lines. This is older code so we don't need to update the rest of the file, but don't need to introduce them here.